packages/app/src/themes/componentOverrides.ts

```diff
@@ -68,16 +87,137 @@ export const components = (
         },
       },
     },
+    BackstageSidebarItem: {
+      styleOverrides: {
+        label: {
+          '&[class*="MuiTypography-subtitle2"]': {
+            fontWeight: '500',
+          },
+        },
+      },
+    },
+    BackstagePage: {
+      styleOverrides: {
+        root: {
+          overflow: 'scroll',
+        },
+      },
+    },
     BackstageContent: {
       styleOverrides: {
         root: {
           backgroundColor: themePalette.general.mainSectionBackgroundColor,
           '& div:first-child': {
             '& > div[class*="-searchBar"]': {
               backgroundColor: themePalette.general.formControlBackgroundColor,
-              boxShadow: `0px 2px 1px -1px ${themePalette.general.disabled}`,
+              border: `1px solid ${themePalette.general.searchBarBorderColor}`,
+              boxShadow: 'none',
             },
           },
+          '& > div[class*="-MuiGrid-root"]': {
+            marginLeft: '0',
+            width: '100%',
+          },
+          '& > div[class*="v5-MuiGrid-root v5-MuiGrid-container v5-MuiGrid-spacing-xs-3"] > div.v5-MuiGrid-item:nth-child(odd)':
+            {
+              paddingLeft: '0',
```

Comment:
Hi @debsmita1 This line is for alignment of cards on Settings page. Can you confirm again if it works for you? Also good catch on the Learning Path alignment 👍 Will update the PR. Thank you.

Reply:
Hi @debsmita1 , I have updated the selector to make it more universal to locate these cards, can you confirm if it works for you now?